test/src/test/java/org/apache/accumulo/test/proxy/SimpleProxyBase.java

```diff
@@ -1362,6 +1364,82 @@ public void userPermissions() throws Exception {
   }
 
   @Test
+  public void testNamespaceOps() throws Exception {
+    String testNamespace = "testnamespace";
+    String testNamespace2 = "testnamespace2";
+    assertEquals("", client.defaultNamespace(creds));
+    assertEquals("accumulo", client.systemNamespace(creds));
+    client.createNamespace(creds, testNamespace);
+    assertTrue(client.namespaceExists(creds, testNamespace));
+    assertTrue(client.listNamespaces(creds).contains(testNamespace));
```

Comment:
Would be good to also check the size of the returned list.  This will make the test catch unexpected junk in the returned list.